src/history/hs.c

```diff
@@ -10,9 +10,9 @@
 
 /*
  * When an operation is accessing the history store table, it should ignore the cache size (since
- * the cache is already full), and the operation can't reenter reconciliation.
+ * the cache is already full).
  */
-#define WT_HS_SESSION_FLAGS (WT_SESSION_IGNORE_CACHE_SIZE | WT_SESSION_NO_RECONCILE)
+#define WT_HS_SESSION_FLAGS WT_SESSION_IGNORE_CACHE_SIZE
```

Comment:
Should we just replace WT_HS_SESSION_FLAGS with WT_SESSION_IGNORE_CACHE_SIZE

Reply:
We could, we would have to move the above comment to a relevant location to make sure the context isn't lost. I don't see particular value in doing so however. Keeping them separate may enable future flags to be set with ease?

Reply:
That's fair.